src/wallet.js

```diff
@@ -195,17 +213,22 @@ const wallet = {
    * @param {string} xpriv Extended private-key to start wallet
    * @param {string} pin
    * @param {boolean} loadHistory if should load the history from the generated addresses
+   * @param {object} multisig MultiSig data including minSignatures and xpubs of participants
    *
    * @return {Promise} Promise that resolves when finishes loading address history, in case loadHistory = true, else returns null
    * @memberof Wallet
    * @inner
    */
-  executeGenerateWalletFromXPriv(xprivkey, pin, loadHistory) {
+  executeGenerateWalletFromXPriv(xprivkey, pin, loadHistory, multisig) {
     const xpriv = HDPrivateKey(xprivkey);
     let initialAccessData;
     let privkey;
     if (xpriv.depth === 0) {
-      privkey = xpriv.deriveNonCompliantChild(`m/44'/${HATHOR_BIP44_CODE}'/0'/0`);
+      if (multisig) {
+        privkey = xpriv.deriveNonCompliantChild(`m/45'/${HATHOR_BIP44_CODE}'/0'/0`);
```

Comment:
Can we use the constant + `/0`?

Reply:
Done in [89d3a89](https://github.com/HathorNetwork/hathor-wallet-lib/pull/324/commits/89d3a89fb831ed94006445e3d29178170df31c29)